mrp_component_operation/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "MRP Components Operations",
-    "version": "14.0.1.2.0",
+    "version": "17.0.1.2.0",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "summary": "Allows to operate the components from a MO",
     "website": "https://github.com/OCA/manufacture",
```

Comment:
Down in dependencies, we need to wait until stock_restrict_lot dependency is solved (https://github.com/OCA/stock-logistics-workflow/pull/1840 ) to change or not the dependency of stock_move_forced_lot to stock_restrict_lot.

Reply:
Already merged (https://github.com/OCA/stock-logistics-workflow/pull/1849) please, change the dependency and adapt the methods

Reply:
Changes applied!